app/views/admin/environments/index.html.erb

```diff
@@ -21,7 +21,7 @@
             <td>
               <%= link_to "Edit", edit_admin_environment_path(env) %>
               |
-              <%= link_to "Delete", admin_environment_path(env), method: :delete, data: { confirm: "Are you sure?" } %>
+              <%= link_to_delete(admin_environment_path(env)) %>
```

Comment:
avoiding adding any `options` [here](https://github.com/rails/rails/blob/master/actionpack/lib/action_dispatch/routing/polymorphic_routes.rb#L99) so just updating links to `#index`